src/core/ColorUtil.cc

```diff
@@ -100,4 +103,194 @@ std::optional<Color4f> parse_hex_color(const std::string& hex) {
   return rgba;
```

Comment:
As long as you're nearby, a few lines up on old line 96 there is a comment that says
```
// convert the hex character(s) from base 16 to base 10
```
The part about base 10 is not at all correct; there is no base 10 anywhere in the picture.  It's converting to integer (a binary format) and thence to floating point (also a binary format).

Reply:
Best might be "to a fraction between 0 and 1 inclusive".